scripts/examples/03-Machine-Learning/00-TensorFlow/face_landmarks_multi_face.py

```diff
@@ -0,0 +1,57 @@
+# This work is licensed under the MIT license.
+# Copyright (c) 2013-2025 OpenMV LLC. All rights reserved.
+# https://github.com/openmv/openmv/blob/master/LICENSE
+#
+# This example shows off Google's MediaPipe Face Landmark Detection model for multiple faces.
+#
+# NOTE: This exaxmple requires an OpenMV Cam with an NPU like the AE3 or N6 to run real-time.
+
+import csi
+import time
+import ml
+from ml.preprocessing import Normalization
+from ml.postprocessing.mediapipe import BlazeFace
+from ml.postprocessing.mediapipe import FaceLandmarks
+
+# Initialize the sensor.
+csi0 = csi.CSI()
+csi0.reset()
+csi0.pixformat(csi.RGB565)
+csi0.framesize(csi.VGA)
+csi0.window((400, 400))
+
+# Load built-in face detection model
+face_detection = ml.Model("/rom/blazeface_front_128.tflite", postprocess=BlazeFace(threshold=0.4))
+print(face_detection)
+
+# Load built-in face landmark model
+face_landmarks = ml.Model("/rom/face_landmarks_192.tflite", postprocess=FaceLandmarks(threshold=0.4))
+print(face_landmarks)
+
+clock = time.clock()
+while True:
+    clock.tick()
+    img = csi0.snapshot()
+
+    # faces is a list of ((x, y, w, h), score, keypoints) tuples
+    faces = face_detection.predict([img])
+
+    if faces:
+        for r, score, keypoints in faces[0]:
```

Comment:
```suggestion
    # Faces is a list of ((x, y, w, h), score, keypoints) tuples
    for r, score, keypoints in face_detection.predict([img]):
```
This doesn't return None.